docs/shape_batch_size_and_layout.md

```diff
@@ -29,3 +29,12 @@ it ignores the batch_size value.
 on [Shape Inference Document](https://docs.openvinotoolkit.org/latest/_docs_IE_DG_ShapeInference.html).
 In case the model can't be reshaped, it will remain in the original parameters and all requests with incompatible input format
 will get an error. The model server will also report such problem in the logs.
+
+# Changing model input/output layout
+- Layout parameter is optional. By default layout is inherited from OpenVINO™ model. You can specify layout during conversion to IR format via Model Optimizer.
```

Comment:
How about ONNX models? Do this whole feature apply to them as well? 
Can you mention it somewhere in the docs?

Reply:
Yes, this also works for ONNX models.

Reply:
Mentioned